CDTDatastore/touchdb/TD_Database.m

```diff
@@ -582,7 +582,9 @@ + (BOOL)deleteClosedDatabaseAtPath:(NSString *)path error:(NSError **)outError
         NSString *attachmentsPath = [TD_Database attachmentStorePathWithDatabasePath:path];
 
         success =
```

Comment:
It might be worth adding a comment about what files are being removed from where.